lib/matplotlib/dviread.py

```diff
@@ -526,8 +527,7 @@ class DviFont(object):
     __slots__ = ('texname', 'size', 'widths', '_scale', '_vf', '_tfm')
 
     def __init__(self, scale, tfm, texname, vf):
-        if six.PY3 and isinstance(texname, bytes):
-            texname = texname.decode('ascii')
+        assert(isinstance(texname, bytes))
```

Comment:
No parentheses here.  (And in general I think we should avoid bare asserts in favor of raising explicit exceptions.)